test/drivers/test_driver_imports.py

```diff
@@ -0,0 +1,51 @@
+# import unittest.mock
+# import sys
+# import os
+# import ctypes
+import pytest
+
+
+def test_driver_files():
+    import fixate.drivers.dmm
+    import fixate.drivers.dso
+    import fixate.drivers.funcgen
+    import fixate.drivers.lcr
+    import fixate.drivers.pps
+
+
+# _files_path = os.path.join(os.path.dirname(__file__), "files")
```

Comment:
If we're not going to use this, I think we should delete the commented out code and remove the .h/.dll files.

Could keep this on a separate branch if you want to keep it in the repo. Hopefully Jasper's work will replace the need for all this soon enough in any case.

Reply:
Done